packages/nc-gui/composables/useColumnCreateStore.ts

```diff
@@ -82,8 +82,6 @@ const [useProvideColumnCreateStore, useColumnCreateStore] = createInjectionState
     const { resetFields, validate, validateInfos } = useForm(formState, validators)
 
     const onUidtOrIdTypeChange = () => {
-      const { isCurrency } = useColumn(ref(formState.value as ColumnType))
```

Comment:
This causes problems as `useColumn` uses `inject` which it can't trigger in a callback - `provide` and `inject` can only be used inside the body of a setup function

So `isCurrency.value` probably is always false currently (also there's a double check for it below